docs/src/www/index.html

```diff
@@ -9,6 +9,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" type="text/css" href="css/main.css">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="shortcut icon" href="/images/favicon.ico">
```

Comment:
What about adding the favicon at the root level. `href="/favicon.ico">` that's where the browser is looking for it when the link isn't set.